src/app/ubs/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-location-dashboard.component.html

```diff
@@ -214,6 +214,50 @@
       </mat-menu>
     </tr>
   </tbody>
+  <tbody *ngIf="!showAllTariff && !isLoading">
+    <tr class="selected-card">
+      <td>
+        {{ selectedCard.region }}
+      </td>
+      <td>
+        <div *ngFor="let item of selectedCard.city">
+          {{ item }}
+        </div>
+      </td>
+      <td>
+        <div>
+          {{ selectedCard.courier }}
+        </div>
+      </td>
+      <td>
+        <div *ngFor="let item of selectedCard.station">
+          {{ item }}
+        </div>
+      </td>
+      <td>
+        {{ selectedCard.tariff | tariffStatus }}
+      </td>
+      <td class="last-col" *ngIf="selectedCard.tariff != 'Неактивно'" [matMenuTriggerFor]="crumbs">
```

Comment:
It's not good to use translation in if statement. You should use either enum or a specific key. Also replace != with !==.